design-proposals/nic-hotplug/nic-hotplug.md

```diff
@@ -143,6 +143,42 @@ The proposed API changes for VM objects can be seen in
 [the VM API examples section](#vms), while the proposed API changes for the VMI
 object can be seen in [the VMI API examples section](#vmis).
 
+### Pod interface naming
+As of today, the VMs’ pod interfaces names has an ordinal factor 
+(`net1`, `net2`, …, `netX`) which is derived from the specified in the VMI spec,
+the interfaces names are specified in the virt-launcher pod 
+`k8s.v1.cni.cncf.io/networks` annotation, and eventually the pod is created with 
+those interface names.
+
+Given a VMI with 2 secondary interfaces its pod `k8s.v1.cni.cncf.io/networks` 
+annotation would have `net1, net2, net3` elements.
+When removing interface `net2`, the annotation will change to `net1`, `net3`, 
+which makes associating the pod to VM interfaces impossible.
+
+In order to enable unplug functionality virt-launcher pod network interfaces names 
+shall be changed in a way to won't have an ordinal factor.
+
+The pod interface name will be derived from the `kubevirt-spec-iface-name`; 
+we'll simply compute an Hash of the interface name
+(which is guaranteed to be unique within each VMI), and ensure all generated
+names for the pod's networking infrastructure are accepted by the kernel.
```

Comment:
Could you please add a note explaining the requirements for the name to be accepted by the kernel?